### DISCUSSION TITLE
Undefined constant "Spatie\Backup\Commands\SIGINT"

### DISCUSSION THREAD

**Discussion by pedami** [spatie/laravel-backup#1565] [Q&A]
Hola, yo tengo el mismo problema, la extensión está instalada correctamente y el entorno está en producción con la última versión de Laravel Nova:

[2022-08-13 04:12:13] local.ERROR: Undefined constant "Spatie\Backup\Tasks\Backup\SIGINT" {"userId":1,"exception":"[object] (Error(code: 0): Undefined constant "Spatie\Backup\Tasks\Backup\SIGINT" at /home/*************************/public_html/vendor/spatie/laravel-backup/src/Tasks/Backup/BackupJob.php:148)

Gracias.

**Comment by pedami** [comment]
I have activated pcntl, but it still does not go, I have seen that we may have to activate some more functions of this extension, could you tell me which of them?

Thanks.

![image](https://user-images.githubusercontent.com/6138775/184730110-62848696-4133-482d-943b-51e13b0c3359.png)

**Reply by angeljqv** [reply]
All

**Comment by DevHoracioRodriguez** [comment]
Did you solved this?

Anyways, [here](https://github.com/shuvroroy/filament-spatie-laravel-backup/issues/20#issuecomment-1165244914) is something that I tried.

I ended up with the following code and seems to work.

`    public function __construct($option = '')
    {
        \Artisan::call('backup:run' . $this->option);
    }`

Do you have any suggestions?
